### PR TITLE
how to v2

### PR DESCRIPTION
**Pull request by kilrau**
less text, more overview

### CODE REVIEW

File: How-to-contribute.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
+Yay! Kudos for being a part of Exchange Union & help building XUD, our node software! Any contributions are welcome no matter how big or small. From typos and bug fixes to entire modules - you will be awarded with XUC, our token.
 
-If you'd like to contribute to the Exchange Union, please fork our repository, push your commits or fixes and send a pull
-request for us to review and merge into the development branch. If, however, you wish to submit substaintial changes or features -
-We encourage you to first have a chat with us on our [Slack Channel](https://exchangeunion.slack.com/).
+Here is how it works:
+1. Grab a task from [here.](https://github.com/ExchangeUnion/xud/issues)
+2. Each task carries a range of XUC to be earned, like e.g. 50-100 XUC. The higher the reward, the more complex - grab a task which best fits your time at hand & skill set.
+3. Let everyone know via the comment field (!) and fork the repo to start workin on your task.
```
Comment (moshababo) [inline How-to-contribute.md]: Let everyone know 'by commenting'

File: How-to-contribute.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
 
-If you'd like to contribute to the Exchange Union, please fork our repository, push your commits or fixes and send a pull
-request for us to review and merge into the development branch. If, however, you wish to submit substaintial changes or features -
-We encourage you to first have a chat with us on our [Slack Channel](https://exchangeunion.slack.com/).
+Here is how it works:
+1. Grab a task from [here.](https://github.com/ExchangeUnion/xud/issues)
+2. Each task carries a range of XUC to be earned, like e.g. 50-100 XUC. The higher the reward, the more complex - grab a task which best fits your time at hand & skill set.
+3. Let everyone know via the comment field (!) and fork the repo to start workin on your task. 
+4. [Get in touch](https://gitter.im/exchangeunion/Lobby) for help on the way, especially for testing. Our automation is still in the works :/
+5. When you feel ready, submit a pull request  against the `development` branch with your <b>ether address (!)</b> in the comment section
```
Comment (moshababo) [inline How-to-contribute.md]: Don't see a reason why not to open the PR against `master`

File: How-to-contribute.md (hunk 1 above)
Comment (sangaman) [inline How-to-contribute.md]: Is "workin" intentional or should it be "working?" If you want the informal working, might as well stick an apostrophe at the end, but I prefer "working."

Also it's not clear what to let everyone know, maybe it's more clear to say "Comment to let everyone know you'll be working on the task, and fork the repository to get started."

File: How-to-contribute.md [hunk 3]
```diff
@@ -1,22 +1,20 @@
-<h1> HOW TO CONTRIBUTE </h1>
+<h1> How To Contribute & Earn XUC </h1>
 
-Thank you for considering to help in building Exchange Union! We welcome any contributions no matter how big or small - from typos to bug fixes, you will be awarded with XUC (Exchange Union's token) after carefully analyzing every pull request. Github is our git version control service of choice, used to track issues, suggestions, feature request, documentations and is home to the source code which powers Exchange Union. 
+Yay! Kudos for being a part of Exchange Union & help building XUD, our node software! Any contributions are welcome no matter how big or small. From typos and bug fixes to entire modules - you will be awarded with XUC, our token.
```
Comment (sangaman) [inline How-to-contribute.md]: "helping build" instead of "help building."

File: How-to-contribute.md (hunk 2 above)
Comment (sangaman) [reply How-to-contribute.md]: Yes, it should be against master.

File: How-to-contribute.md [hunk 4]
```diff
@@ -1,22 +1,20 @@
...
-We encourage you to first have a chat with us on our [Slack Channel](https://exchangeunion.slack.com/).
+Here is how it works:
+1. Grab a task from [here.](https://github.com/ExchangeUnion/xud/issues)
+2. Each task carries a range of XUC to be earned, like e.g. 50-100 XUC. The higher the reward, the more complex - grab a task which best fits your time at hand & skill set.
+3. Let everyone know via the comment field (!) and fork the repo to start workin on your task. 
+4. [Get in touch](https://gitter.im/exchangeunion/Lobby) for help on the way, especially for testing. Our automation is still in the works :/
+5. When you feel ready, submit a pull request  against the `development` branch with your <b>ether address (!)</b> in the comment section
+6. For non-trivial changes, expect some review rounds and quite some testing to take place before your changes get merged into master
+7. After a successful merge into master we'll send you your XUC. For now this is all manual work on our end, but pleeease develop a smart contract with github oracles to automate this whole process (propose a price tag!)
```
Comment (sangaman) [inline How-to-contribute.md]: This sort of automation can be done without smart contracts too, probably less risky that way.

**Review by sangaman — COMMENTED** [review]
I think we should put a CONTRIBUTING.MD in our main XUD repo as well, as is common with open source projects. Maybe a logical approach is to have this guide focus on how to earn XUC and participate in the project, whereas the CONTRIBUTING.MD file covers nitty gritty that should apply to all contributions. I can propose a guide there myself, just wanted to check for any feedback before I start.

File: How-to-contribute.md (hunk 2 above)
Comment (kilrau) [reply How-to-contribute.md]: ok, changed

File: How-to-contribute.md (hunk 1 above)
Comment (kilrau) [reply How-to-contribute.md]: Haha, not exactly intentional. Supposed to be casual, but not too gangsta..

File: How-to-contribute.md (hunk 3 above)
Comment (kilrau) [reply How-to-contribute.md]: Fixed

File: How-to-contribute.md (hunk 4 above)
Comment (kilrau) [reply How-to-contribute.md]: True, but we would be in control of that solution/wallet where the XUC come from whereas with a smart contract we wouldn't be. Anyhow I changed it to a more generic expression.

**Pull request merged**